docs/adr/0003-2021-02-24-when-to-adr.md

```diff
@@ -0,0 +1,58 @@
+# When to create an ADR
+
+## Status
+
+Proposed
+
+## Context
+
+As a team, we weren't sure when decisions should be recorded in an Architectural Decision Record (ADR). 
+
+However, we know that we should record certain decisions made on this project, so that others can more easily understand the motivation behind decisions that have been made.
+
+ADRs will also be useful for the GDS technical assessment, so we decided to [get together](https://miro.com/app/board/o9J_lZuM9qs=/?moveToWidget=3074457354907364156&cot=14) to agree on when and how to write them.  
+
+## Decision
+
+We should consider writing an ADR when we find ourselves making decisions that require in-depth discussions.
+
+Since this is a new way of documenting for a few of us, if we're unsure whether an ADR is appropriate, we should reach out to the rest of the development team on Slack to see whether they feel that there's value in writing one.
```

Comment:
I think this might read a little bit better if you bulleted the steps, so:

We should:

- Reach out to the rest of the development team on Slack to see whether they feel there's value in writing one
- The rest of the team should respond with emojis or in the thread to help the team reach a decision